engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-environment-modal.html

```diff
@@ -63,6 +58,10 @@ <h4 translate>Assign Lifecycle Environment and Content View</h4>
   </div>
 
   <div data-block="modal-footer">
+    <p style="text-align:left;font-size:12px;"><b>Note:</b> Updating the content view here will not automatically update the client. </p>
```

Comment:
I don't think we need to word 'Note' - it's already in a tooltip :)

Reply:
True, removed :)